lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1187,6 +1198,14 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => deprecated_description(:'instrumentation.curb', 'If `true`, disables instrumentation for the curb gem.')
         },
+        :disable_custom_events_instrumentation => {
+          :default => false,
+          :public => true,
+          :type => Boolean,
+          :dynamic_name => true,
```

Comment:
Same situation as the other config with dynamic_name, we should be able to remove this line since you're calling the config for this directly in the custom_events file

Reply:
Updated with https://github.com/newrelic/newrelic-ruby-agent/pull/1659/commits/951476b03bd2beafa3f9804a04b79b0b8a88ba6f